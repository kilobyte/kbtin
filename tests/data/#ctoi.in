#alias ctoi {#ctoi x $0;#showme $x}
ctoi 六十
ctoi 二十
ctoi 二百
ctoi 兩百
ctoi 兩千
ctoi 四十五
ctoi 兩千三百六十二
ctoi 十四
ctoi 一萬二千
ctoi 一百一十四
ctoi 一千一百五十八
ctoi 零
