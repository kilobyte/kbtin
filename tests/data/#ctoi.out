60
20
200
200
2000
45
2362
14
12000
114
1158
0
